code/game/gamemodes/cult/cult.dm

```diff
@@ -164,4 +168,97 @@
 			devoted to stopping this cult. Note that holy water seems to weaken and eventually return the minds of cultists that ingest it, and mindshield implants will prevent conversion \
 			altogether."
 
+/datum/game_mode/proc/begin_bloodstone_phase()
+	for(var/i = 0, i < 4, i++) //four bloodstones
+		var/spawnpoint = get_turf(pick(GLOB.generic_event_spawns))
+		new /obj/structure/destructible/cult/bloodstone(spawnpoint)
+	var/list/bloodstone_areas = list()
+	for(var/obj/structure/destructible/cult/bloodstone/B in bloodstone_list)
+		var/area/A = get_area(B)
+		bloodstone_areas.Add(A.map_name)
+	priority_announce("Figments of an eldritch god are being pulled through the veil anomaly in [bloodstone_areas[1]], [bloodstone_areas[2]], [bloodstone_areas[3]], and [bloodstone_areas[4]]! Destroy any occult structures located in those areas!","Central Command Higher Dimensional Affairs")
+	addtimer(CALLBACK(src, .proc/increase_bloodstone_power), 300)
+
+/datum/game_mode/proc/increase_bloodstone_power()
+	if(!bloodstone_list.len) //check if we somehow ran out of bloodstones
+		return
+	for(var/obj/structure/destructible/cult/bloodstone/B in bloodstone_list)
+		if(B.current_fullness == 9)
+			create_anchor_bloodstone()
+			return //We're done here
+		else
+			B.current_fullness++
+		B.update_icon()
+	addtimer(CALLBACK(src, .proc/increase_bloodstone_power), 300)
+
+/datum/game_mode/proc/create_anchor_bloodstone()
+	if(SSticker.mode.anchor_bloodstone)
+		return
+	var/obj/structure/destructible/cult/bloodstone/anchor_target = bloodstone_list[1] //which bloodstone is the current cantidate for anchorship
+	var/anchor_power = 0 //anchor will be faster if there are more stones
+	for(var/obj/structure/destructible/cult/bloodstone/B in bloodstone_list)
+		anchor_power++
+		if(B.obj_integrity > anchor_target.obj_integrity)
+			anchor_target = B
+	SSticker.mode.anchor_bloodstone = anchor_target
+	anchor_target.name = "anchor bloodstone"
+	anchor_target.desc = "It pulses rythmetically with migrane-inducing light. Something is being reflected on every surface, something that isn't quite there..."
+	anchor_target.anchor = TRUE
+	anchor_target.max_integrity = 1200
+	anchor_target.obj_integrity = 1200
```

Comment:
Isnt that a bit.. much

Reply:
It should be doable, especially if the crew has already bashed a few stones before that stage. In the only test where there was competition the crew was able to take it down after breaking the other 3 stones